unix/Makefile

```diff
@@ -36,7 +36,7 @@ endif
 ifeq ($(UNAME_S),Darwin)
 CC = clang
 # Use clang syntax for map file and set osx specific flags
```

Comment:
Maybe remove the "and set osx specific flags" part of this comment since it is no longer relevant?